### DISCUSSION TITLE
Nginx reverse proxy for all-in-one build

### DISCUSSION THREAD

**Discussion by develephant** [Budibase/budibase#7607] [Help]
Hi,

Does the all-in-one build support being behind an nginx proxy? I have been trying for over 6 hours now, and none of the instructions on the site are working. Is there a reason for this?

I would love to promote your product, but there needs to be clearer documentation on how to make this work. Or at least documentation that is current.

I feel like I wasted a day chasing shadows.

**Comment by shogunpurple** [comment]
What does your NGINX config look like?
